client/my-sites/stats/activity-log/style.scss

```diff
@@ -40,6 +40,7 @@
 
 .activity-log__empty-day-title {
 	font-weight: 600;
+	white-space: nowrap;
```

Comment:
I think we should make it clear this is overflowing by adding ellipsis. If you add the following rules to what you've already written, it should just work:

```
overflow: hidden;
margin-right: 24px;
text-overflow: ellipsis;
```

Reply:
hmm…not sure how useful this is: `January 13, 2018 - Jan…`

also, from what I was seeing, if we scrunch the screen small enough that we can't see the full date then it seems like other major parts of the Activity Log break down as well